pypsa/components.py

```diff
@@ -387,10 +390,17 @@ def _build_dataframes(self):
 
             static_dtypes = attrs.loc[attrs.static, "dtype"].drop(["name"])
 
-            df = pd.DataFrame(
-                {k: pd.Series(dtype=d) for k, d in static_dtypes.items()},
-                columns=static_dtypes.index,
-            )
+            if component == "Shape":
+                df = gpd.GeoDataFrame(
+                    {k: gpd.GeoSeries(dtype=d) for k, d in static_dtypes.items()},
+                    columns=static_dtypes.index,
+                    crs="epsg:4326",
```

Comment:
This should probably be taken from `n.srid` or the new `n.crs`, rather than implicitly assuming a projection. Then the default of these should be "EPSG:4326".